lib/api/client-commands/captureNetworkCalls.js

```diff
@@ -0,0 +1,57 @@
+const ClientCommand = require('./_base-command.js');
+const {Logger} = require('../../utils');
+
+/**
+ * Capture outgoing network calls from the browser.
+ *
+ * @example
+ *  this.demoTest = function (browser) {
+ *    browser
+ *      .captureNetworkCalls((requestParams) => {
+ *        console.log('Request URL: ', requestParams.request.url);
+ *        console.log('Request method: ', requestParams.request.method);
+ *        console.log('Request headers: ', requestParams.request.headers);
+ *      })
+ *      .navigateTo('https://www.google.com');
+ *  };
+ *
+ * @method captureNetworkCalls
+ * @syntax .captureNetworkCalls(callback)
+ * @param {function} callback callback function to be called whenever a outgoing network call is made.
+ * @api protocol.userprompts
+ */
+class CaptureNetworkCalls extends ClientCommand {
+
+  performAction(callback) {
+
+    if (!this.api.isChrome()  && !this.api.isEdge()) {
+      const error =  new Error('CaptureNetworkCalls is not supported while using this driver');
```

Comment:
can you change this to say in that "is only supported in Chrome or Edge drivers"?